.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Should I file this?**
+Use this template to report *observed bugs in current Plotman*.  Do not use this to request tech support or troubleshooting; we are unable to provide that support here, so please use the user channel(s) on Keybase, or GitHub Discussions instead.  Do not use this to report issues with the core Chia plotter.  If you are using `main` branch, try pulling `development` to see if your problem is already fixed.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior, e.g.:
+1. Set up config with '...'
+2. Run other programs '....'
+3. Run Plotman with '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
```suggestion
A clear and concise description of what you expected to happen, even if it seems obvious. 
 Sometimes there are multiple aspects that are 'obvious' and it helps us to know which you are referring to.
```

Side note, I do usually like [semantic newlines](https://sembr.org/).  Something to consider as an overall policy.  Though I can't say I'm actually sure here since we know the editor will wrap and we know that people will be viewing it as markdown source, not rendered html.

Reply:
Semantic newlines sounds reasonable; I'll adopt that in general.